scopes/lanes/lanes/lane.cmd.ts

```diff
@@ -356,12 +359,71 @@ https://${docsDomain}/components/lanes`;
   }
 }
 
+export class LaneReadmeRemoveCmd implements Command {
+  name = 'readme-remove [laneName]';
+  description = 'remove lane readme component';
+  options = [] as CommandOptions;
+  loader = true;
+  private = true;
+  skipWorkspace = false;
+
+  constructor(private lanes: LanesMain) {}
+
+  async report([laneName]: [string]): Promise<string> {
+    const { result, message } = await this.lanes.removeLaneReadme(laneName);
+
+    if (result) {
+      return chalk.green(
+        `the readme component has been successfully from the lane ${laneName || this.lanes.getCurrentLane()}`
```

Comment:
the work "removed" is probably missing here: 

the readme component has been *removed* successfully